### ISSUE TITLE
Fail to decode any meaningful output.

### ISSUE THREAD

**Issue by rylanchiu** [abisee/pointer-generator#34]
I have trained the model for about 80,000 iterations and the loss has decreased to about 0.000004~, which is really low. But when I run the model in decode mode, the output is just `what what what what...` or `why why why why...`. I am using the quora data set [here](https://www.kaggle.com/c/quora-question-pairs/data) for training. There are about 150,000 pairs duplicate questions. I wonder do you have similar experience in your training and how do you fix it? Thanks!

**Comment by rylanchiu** [comment]
I found that the data I used did not have any `SENTENCE END` symbol. Now I fixed this issue and make sure that the input in the `run_train_step` are all good. I run the model 80,000 iterations again. But the loss fail to decrease normally. Do you have idea about it? Thanks in advance!

**Comment by abisee** [comment]
Hi @rylanchiu, glad to hear you resolved the `SENTENCE END` issue. If the loss is not decreasing, a standard approach is to try reducing your learning rate, or maybe try a different [optimization algorithm](https://github.com/abisee/pointer-generator/blob/master/model.py#L300).

Have you visualized the attention distribution with the [attention visualizer tool](https://github.com/abisee/attn_vis)? For our work, we found this was more useful than anything else, in trying to understand the results.

**Comment by rylanchiu** [comment]
I found the issue is caused by the data reading. Now I fixed it. Everything works well now. Thanks!

**Comment by Shujian2015** [comment]
Hi @rylanchiu, I am facing the similar issue. Could you clarify how you solved the data reading problem? Thanks.

**Comment by DivyaWadehra** [comment]
I am also getting the same problem .. can you hep me on this?
